modules/default.nix

```diff
@@ -1,4 +1,4 @@
-{ niv, cbspkgs, pkgs }:
+{ niv }:
 {
-  hydra = "${niv.hydra}/hydra-module.nix";
+  hydra = import "${niv.hydra}/hydra-module.nix";
 }
```

Comment:
what exactly changes for the users of a module here? 

currently, it's used like this:

```
imports = [
    cbspkgs.modules.hydra
};
```

Reply:
i found it more sound to have all modules as lambdas than having some to be lambdas, some to be nix paths. but this is rather unimportant.

Reply:
just to be clear, how would this be used in a nixos configuration then?

Reply:
From an offline discussion:

Nothing at all will change in how the modules are used, because the imports in a nixos configuration are polymorphic.